src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
The `href` attribute is pointing to a `.scss` file. It should point to a compiled `.css` file instead, such as `./styles/index.css`, to ensure the styles are applied correctly.